.travis.yml

```diff
@@ -15,10 +15,12 @@ jobs:
         - sudo mv docker-compose /usr/local/bin
       install:
         - pip install --upgrade pip MySQL-python
-        - ./docker/build-images.sh ${TRAVIS_BRANCH} -p
+        - ./codalab-service.py build bundleserver -v ${TRAVIS_BRANCH} --push
+        - ./codalab-service.py build worker -v ${TRAVIS_BRANCH} --push
+        - ./codalab-service.py build frontend -v ${TRAVIS_BRANCH} --push
         - ./setup.sh server
       script:
-        - ./docker/codalab-service/start-service.sh -i -t -w
+        - ./codalab-service.py start -i -w -t --root-user codalab --root-pwd testpassword --worker-dir /home/travis/codalab/worker-dir --rest-port 2900 --mysql-port 3306 --version ${TRAVIS_BRANCH}
```

Comment:
can we use the long form `--root-password`?

Reply:
Good idea, done